exercises/hello-world/hello_test.go

```diff
@@ -2,25 +2,17 @@ package greeting
 
 import "testing"
 
-// Define a function HelloWorld(string) string.
+// Define a function HelloWorld() string.
```

Comment:
Yeah, better definitely, I was uncomfortable with using signature anyway, can confuse the issue further.